### ISSUE TITLE
Can't reproduce the result.

### ISSUE THREAD

**Issue by jamiechoi1995** [doubledaibo/clcaption_nips2017#4]
Hello Bo Dai,

I use cl_train_adaptiveattention.lua, 
and I found that I can't reproduce the result in your paper.

After the 1st epoch training, I found that the CIDEr score is **lower than** the original model(before training) by ~0.15,
the training loss is about 4.01 and val loss is about 1.40.

As the training continues, training loss goes down slowly to ~3.8
and val loss goes up and down between 1.3~1.4,
although the CIDEr score is improving, it still can't go higher than the original score.

I use the pre-trained model in AdaptiveAtteion as the target model and reference model, resnet-152 as CNN model and learning rate follows the value in your paper, which is 1e-6.

Could you please help me with this?
Thanks.
